homeassistant/components/whirlpool/strings.json

```diff
@@ -51,9 +51,6 @@
         "state": {
           "unknown": "Unknown",
           "empty": "Empty",
-          "25%": "25%",
-          "50%": "50%",
-          "100%": "100%",
```

Comment:
These should not have been removed, you now have values with missing translations.

These should be added:

```
          "25": "25%",
          "50": "50%",
          "100": "100%",
```

Reply:
I see - I think I understand now, until I break it again :)